Wikibase/Sniffs/Commenting/ClassLevelDocumentationSniff.php

```diff
@@ -60,12 +62,31 @@ public function process( File $phpcsFile, $stackPtr ) {
 			return;
 		}
 
+		if ( $this->license ) {
+			$docClose = $phpcsFile->findPrevious( T_DOC_COMMENT_CLOSE_TAG, $stackPtr );
+			$docStart = $phpcsFile->findPrevious( T_DOC_COMMENT_OPEN_TAG, $docClose );
+			$docBlock = $phpcsFile->getTokensAsString( $docStart, $docClose );
+
+			if ( strpos( $docBlock, "@license {$this->license}" ) === false ) {
+				if ( $phpcsFile->addFixableWarning(
+					'No correct license',
+					$previous,
+					'NoLicense'
+				) ) {
+					$phpcsFile->fixer->addContent( $docClose - 2, " * @license {$this->license}\n" );
+				}
+			} else {
+				//TODO Wrong license
+			}
+		}
+
 		$newlines = substr_count(
 			$phpcsFile->getTokensAsString( $previous + 1, $stackPtr - $previous - 1 ),
 			"\n"
 		);
 		if ( !$newlines ) {
 			// Recreating the correct indention is hard, that's why this is not fixable here
+			//TODO: Can be fixed. There is another Sniff that will fix the indentation
```

Comment:
That might be, but I don't want to repeat something like this here in this sniff.